src/subscription_manager/managercli.py

```diff
@@ -687,6 +687,17 @@ def _validate_options(self):
             else:
                 system_exit(ERR_NOT_REGISTERED_CODE, ERR_NOT_REGISTERED_MSG)
 
+        if self.is_registered() and (
+            getattr(self.options, "username", None) or
+            getattr(self.options, "password", None) or
+            getattr(self.options, "token", None) or
+            getattr(self.options, "org", None)
+        ):
+            system_exit(os.EX_USAGE, _(
+                "Error: --username, --password, --token and --org "
+                "can be used only on unregistered system"
```

Comment:
it should be "systems"

Reply:
The main PR also uses 'system'. I don't see how 'systems' makes any difference... If it does, should the string in 'main' be also updated?

Reply:
I agree with Will, it should be either "on an unregistered system" (generic one) or "on unregistered systems" (generic many).

Reply:
Done.